ggml/src/ggml.c

```diff
@@ -5380,15 +5382,17 @@ static struct ggml_tensor * ggml_group_norm_impl(
 struct ggml_tensor * ggml_group_norm(
     struct ggml_context * ctx,
     struct ggml_tensor * a,
-    int n_groups) {
-    return ggml_group_norm_impl(ctx, a, n_groups, false);
+    int n_groups,
+    float eps) {
+    return ggml_group_norm_impl(ctx, a, n_groups, eps, false);
```

Comment:
(Side question) Is `ggml_group_norm` equivalent to `ggml_norm` with `ggml_reshape` to group rows together beforehand and split them back afterwards?

Reply:
I guess yes?